src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,49 @@
+(function(transport) {
+    "use strict";
+
+    function _log(s){
+        transport.send("bramble-console", s);
+    }
+
+    // Create Console if not present
+    if (!window.console) {
```

Comment:
This has been changed as requested in the newest commit. 
Useful website too, thanks for linking that!